projects/driver_controls/src/main.c

```diff
@@ -3,13 +3,15 @@
 
 #include "adc.h"
 #include "gpio_it.h"
+#include "i2c.h"
 #include "interrupt.h"
 
 #include "analog_io.h"
 #include "digital_io.h"
 #include "event_arbiter.h"
 #include "input_event.h"
 
+#include "ads1015.h"
```

Comment:
Your driver should still build in x86 because you have your stub functions. It just won't do anything.

Reply:
I tried but it wouldn't recognize I2C. 
I had something like I2C_PORT_2 and it would throw an error.
I double checked. "i2c.h" is included in both since it is in ads1015.h.
Is there something I'm missing?

Reply:
You need to add some I2C port stubs to `i2c_mcu.h` for x86. The idea is that for our actual projects, we'll have x86 and STM32 configs that are different, but for now we're just adding stubs.